go-sqlsmith/types/table.go

```diff
@@ -7,12 +7,16 @@ import (
 
 // Table defines database table
 type Table struct {
-	DB string
-	Table string
+	DB          string
+	Table       string
 	OriginTable string
-	Type string
-	Columns map[string]*Column
-	Indexes []string
+	Type        string
+	Columns     map[string]*Column
+	Indexes     []string
+	// Online is for self obtain
+	Online      bool
+	// OnlineOther is for other instances obtain
+	OnlineOther bool
```

Comment:
what are the meaning of `Online` and `OnlineOther`? 😆

Reply:
`Online` means this table is online and will be manipulated in a txn, `OnlineOther` means this table is manipulated in other transactions which should not be a DDL table.